doc/release/release_0.16.rst

```diff
@@ -0,0 +1,386 @@
+Announcement: scikit-image 0.16.0
+=================================
+
+We're happy to announce the release of scikit-image v0.16.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+Starting from this release, scikit-image will follow the spirit of the recently
+introduced numpy deprecation policy -- NEP 29
+(https://github.com/numpy/numpy/blob/master/doc/neps/nep-0029-deprecation_policy.rst). 
+Accordingly, scikit-image 0.16 drops the support for Python 3.5.
+This release of scikit-image officially supports Python 3.6 and 3.7.
+
+New Features
+------------
+- Introduced a new `skimage.evaluate` module containing simple metrics (mse,
+  nrme, psd) and segmentation metrics (adapted rand error, variation of
+  information) (#4025)
+- Added TV-L1 optical flow algorithm for registration --
+`skimage.registration.optical_flow_tvl1` (#3983)
+- Added a function drawing a line in n-dimensional space -- `skimage.draw.line_nd`
+  (#2043)
+- Added Farid & Simoncelli edge filters - `skimage.filters.farid`,
+  `skimage.filters.farid_h`, and `skimage.filters.farid_v` (#3775)
+- Added Majority voting filter assigning to each pixel the most occuring value
+  within its neighboor -- `skimage.filters.majority` (#3836, #3839)
+- Added multi-level threshold "multi-Otsu" method, a thresholding algorithm
+  used to separate the pixels of an input image into several classes by
+  maximizing the variances between classes --
+  `skimage.filters.threshold_multiotsu` (#3872, #4174)
+- Added new data -- `skimage.data.shepp_logan_phantom`, `skimage.data.colorwheel`,
+  `skimage.data.brick`, `skimage.data.grass`, `skimage.data.roughwall`, `skimage.data.cell`
+  (#3958, #3966)
+- Added function to find labeled image properties and convert them to
+  dictionnary -- `skimage.measure.regionprops_table` (#3959)
+- Added new function to convert a polygon into a mask -- `skimage.draw.poly2mask`  (#3971, #3977)
+- Added new visual image comparison helper `skimage.util.compare_images`,
+  that returns an image showing the difference between two input images (#4089)
+- Added new function `skimage.transform.wrap_polar` to remap image into
```

Comment:
wrap -> warp

Reply:
Fun fact… that's a mistake I did in my job applications and that was pointed out to me during my job interview.